src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(number1) {
+    count++;
+
+    return function(number2) {
+      if (count < 4) {
```

Comment:
implement correct logic
<img width="838" alt="image" src="https://user-images.githubusercontent.com/53611910/205438237-76f1738a-bead-40e6-b3d2-03e17c5a0e0c.png">

Reply:
all test corect what i need fix?

Reply:
i all fix and made pull